src/core.js

```diff
@@ -159,3 +149,8 @@ jQuery.fn.size = function() {
 	migrateWarn( "jQuery.fn.size() is deprecated; use the .length property" );
 	return this.length;
 };
+
+jQuery.parseJSON = function() {
+	migrateWarn( "jQuery.parseJSON is deprecated; use JSON.parse" );
```

Comment:
Continuing that train of thought (and related to #152), it would seem the only workarounds/warnings that were present in Migrate 1.x and should still be present in Migrate 3 should be those that concern APIs deprecated before jQuery 1.12/2.2 but not removed in jQuery 1.12/2.2. For the rest, if someone is upgrading from jQuery 1.12/2.2 to jQuery 3.0, they already have them fixed (or their site is already broken).